deny.toml

```diff
@@ -43,7 +43,6 @@ copyleft = "deny"
 
 [[licenses.clarify]]
 name = "ring"
-version = "*"
```

Comment:
The reason this needed removing is somewhat counterintuitive: 

In cargo, `version = "*"` does not in fact mean "any version", but instead "any version that is published on crates.io". We need to remove this (optional) key to truly check any version (in our case our vendored version).